taskPage1.css

```diff
@@ -0,0 +1,80 @@
+.pending-task {
+    height: 80px;
+}
+
+.task-box{
+    padding: 5px 0;
+    height: 80px;
+    width:90%;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    justify-content: space-between;
+}
+
+.task-name{
+    font-style: normal;
+    font-weight: 400;
+    font-size: 23px;
+    color: #000000;
+}
+
+.task-track {
+    text-align: center;
+    max-width: 80px;
+    background: #FFA8A7;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5693010)